src/Components/Card/Card.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import CardHeader from "./CardHeader";
+import CardBody from "./CardBody";
+import './CardStyles.css';
+import CardHeaderEditable from "./CardHeaderEditable";
+import CardBodyEditable from "./CardBodyEditable";
+
+const Card = (props) => {
+
+    const [isEdit, setEdit] = useState(false);
+    const [header, setHeader] = useState(props.content.header);
+    const [body, setBody] = useState(props.content.body);
+    const [save, setSave] = useState(false);
+
+    const editHandler = () => {
+        setEdit(!isEdit);
+        setSave(false);
+    };
+
+    const getHeader = (event) => {
```

Comment:
Be careful with names. If you have some function with name getSomething() then it is expected that function gets something and return it. But your function sets value instead of get